src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -156,6 +156,17 @@ protected function resolveStubPath($stub)
                         : __DIR__.$stub;
     }
 
+    /**
+     * Get the default namespace for the class.
+     *
+     * @param  string  $rootNamespace
+     * @return string
+     */
+    protected function getDefaultNamespace($rootNamespace)
+    {
+        return is_dir(app_path('Models')) ? $rootNamespace.'\\Models' : $rootNamespace;
```

Comment:
What if the directory isn't called that. The other common directory name is `Model`, without the s. Symfony users would certainly namespace with singular words.

Reply:
🤷‍♂️ We have to have some convention.

Reply:
See; Policies, Providers, Scopes, Observers, Exceptions folders.

Reply:
a stat syscall for every getDefaultNamespace() call just to auto-load models in a directory which *might* exist? What about a config instead so we don't need to test it every time?

Reply:
Could be a config in the app config file as well to say what a projects "models" folder is.

Reply:
I think having a config option is unnecessary. This is about pleasing as many people as possible with as few changes as possible. How many people use directories other than `app/` and `app/Models`?

Reply:
i think config option is unneccessary too but to play devils advocate - the config option COULD be models and someone could overwrite it - this will 'please many people as well' and also those who may have called their model folder something  else..